lisa/util/__init__.py

```diff
@@ -231,3 +234,22 @@ def is_valid_url(url: str, raise_error: bool = True) -> bool:
 
 def filter_ansi_escape(content: str) -> str:
     return __ansi_escape.sub("", content)
+
+
+def dump_file(file_name: Path, content: Any) -> None:
+    local_path_parts = constants.RUN_LOCAL_PATH.parts
+    parts_len = len(local_path_parts)
+    run_path = Path(Path()/local_path_parts[parts_len-5] /
```

Comment:
What's this for? I believe it's not necessary. But if it's really needed, it needs comments to explain. In general, to check a path is under another path, you can do below step,

1. call `absolute` to get the absolute path. What ever path, it will be an absolute path.
2. call `relative_to(constants.RUN_LOCAL_PATH)`. It will return the relative path to `constants.RUN_LOCAL_PATH`. If it's not under that path, there is an ValueError. It's ok, if there is an ValueError raise out. Let higher level code to handle it.